Travel Scheduler/Test/Commute.m

```diff
@@ -16,7 +16,9 @@ @implementation Commute
 - (instancetype)initWithOrigin:(NSString *)originId toDestination:(NSString *)destinationId withDepartureTime:(int)departureTime
 {
     self = [super init];
-    self.departureTimeInSecondsSince1970 = departureTime;
+//    self.departureTimeInSecondsSince1970 = departureTime;
```

Comment:
Remove commented code if not needed

Reply:
I sort of do need that line. Was just testing the code by overriding that value in the next line. But I will add a TESTING comment to make that clear!